CMakeLists.txt

```
@@ -55,6 +55,8 @@ endif()
 
 add_library(common-compile-settings INTERFACE)
 
+target_compile_definitions(common-compile-settings INTERFACE NOMINMAX)
```

nit: Are we able to put this inside an if-else for windows?

```suggestion
if (WIN32)
     target_compile_definitions(common-compile-settings INTERFACE NOMINMAX)
endif()
```

---

My mistake. Thought you were trying to get this in tonight.